test/lib/next-test-utils.js

```diff
@@ -104,14 +104,19 @@ export function runNextCommand(argv, options = {}) {
     __NEXT_TEST_MODE: 'true',
   }
 
+  const hookArgs = options.hook ? ['-r', options.hook] : []
```

Comment:
Should we call this option `requireHook` to make it more clear or we could add an option for `nodeArgs` instead?